components/script/dom/xmlhttprequest.rs

```diff
@@ -1038,6 +1044,21 @@ impl XMLHttpRequest {
         encoding.decode(&self.response.borrow(), DecoderTrap::Replace).unwrap().to_owned()
     }
 
+    // https://xhr.spec.whatwg.org/#blob-response
+    fn blob_response(&self) -> Root<Blob> {
+        // Step 1
+        if let Some(response) = self.response_blob.get() {
+            return response;
+        }
+        // Step 2
+        let mime = self.final_mime_type().as_ref().map(::std::string::ToString::to_string).unwrap_or("".to_owned());
```

Comment:
nit: `use std::string::ToString` and replace `::std::string::ToString::to_string` here with `ToString::to_string`.